packages/browser/src/tracekit.ts

```diff
@@ -23,6 +23,7 @@ export interface StackTrace {
   url: string;
   stack: StackFrame[];
   useragent: string;
+  original?: string;
```

Comment:
Had to add this, since there is no other way to get the promise "string" otherwise.
See also: https://github.com/getsentry/sentry-javascript/pull/1838/files#diff-c3aa68526c753b964239284a89d48813R1472